core/ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -184,7 +184,14 @@
       ],
       "name": "default",
       "defaultEnv": "default",
-      "description": null
+      "description": null,
+      "commands": [
+        {
+          "commandLine": "mvn clean install -f ${current.project.path}",
+          "name": "newMaven",
```

Comment:
@benoitf we use short command names so that they are nicely displayed in the widget. But agree, newMaven isn't good.